src/debugging/coreclr/CliDockerClient.ts

```diff
@@ -266,6 +266,17 @@ export class CliDockerClient implements DockerClient {
         return id.substring(0, 12);
     }
 
+    public async copy(sourcePath: string, destinationPath: string): Promise<string> {
+        const command = CommandLineBuilder
+            .create('docker', 'cp')
```

Comment:
Doesn't this fail if the directory does not exist? When I tried it did, anyway.

Reply:
The copy will not create a parent dir and will fail, that is 'docker cp' behavior. If you copy the a/b/c/.vsdbg to /remote_debugger. It will create the target folder.

---
In reply to: [400954085](https://github.com/microsoft/vscode-docker/pull/1800#discussion_r400954085) [](ancestors = 400954085)

Reply:
So we'll need to do a `docker run mkdir /remote_debugger`, right?

Reply:
If you need to create a parent directory, yes.

Reply:
Huh, when I tried it last time it didn't work, but it's working now. Oh well.